### ISSUE TITLE
Classes and schools

### ISSUE THREAD

**Issue by LuissForce** [KUJP-code/materials#7]
![Uploading classes.png…]()
I noticed if you add a class, other schools will have the same class but there is no way to identify which school is that class from.
Can you add the school each class is in?, 
another one could be that when you click classes first there is a filter of schools, once you select the school it shows you all the classes of that school.

**Comment by Brett-Tanner** [comment]
What do you mean by 
> other schools will have the same class

That it appears on the class list? If so then yeah, I'll add the school name as well. But you're logged in as an admin so you're scoped to all classes, if it was an SM they'd only see classes at their school so the school name would be redundant.

**Comment by LuissForce** [comment]
yes, as an admin I think we need it on that list, as an SM is not necessary
